src/lib/index.ts

```diff
@@ -39,7 +39,16 @@ class TPAStylePlugin {
     this.replaceRuntimeModule(compiler);
 
     compiler.hooks.compilation.tap(TPAStylePlugin.pluginName, compilation => {
-      compilation.hooks.optimizeChunkAssets.tapAsync(TPAStylePlugin.pluginName, (chunks, callback) => {
+      const pluginDescriptor = isWebpack5
+        ? TPAStylePlugin.pluginName
```

Comment:
are you sure it's not the other way around?

Reply:
Oh, is is the other way around, nice catch, fixing.